### DISCUSSION TITLE
Request: show location names for users on User Management list

### DISCUSSION THREAD

**Discussion by corsacca** [DiscipleTools/disciple-tools-theme#1222] [Ideas and Requests]
Show the location names in the user management list instead of a checkmark.
To make it a one-stop place to go in terms of finding the potential Multiplier to match with a Seeker.

![image](https://user-images.githubusercontent.com/24901539/108977058-7544b900-7688-11eb-8ded-dc99f5bf3bec.png)

**Comment by oldvill** [comment]
This would be super helpful for the Dispatcher!
